inorbit_edge_executor/worker_pool.py

```diff
@@ -143,6 +148,9 @@ def build_worker_from_serialized(self, serialized_worker) -> Worker:
         # NOTE (Elvio): This validation was added for backward compatibility when the pause/resume
         # feature was added
         worker.set_paused(serialized_worker.state.get("paused", False))
+        # Set the MissionTrackingAPI from the context if available
+        if context.mt is not None:
+            worker.set_mt(context.mt)
```

Comment:
Isn't this the same ? Worst case, it's still `None`

```suggestion
        worker.set_mt(context.mt)
```